libr/debug/session.c

```diff
@@ -232,6 +232,13 @@ static RDebugSnap *r_idx_to_snap(RDebug *dbg, ut32 idx) {
 	return NULL;
 }
 
+R_API void r_debug_session_path(RDebug *dbg, const char *path) {
+	if (dbg->snap_path) {
+		free (dbg->snap_path);
+	}
+	dbg->snap_path = strdup (path);
```

Comment:
i think the r_debug_session_path() should expand home using r_str_home and use abspath to avoid issues if the user decides to chdir after setting a relative path here.